### PR TITLE
Fix agentskills tests

### PR DESCRIPTION
**Pull request by tobitege**
Test like "test_edit_file_not_opened" in "test_agent_skill.py" failed, as in it did not raise an exception editing a file, which should not be open in the first place.

Meaning: the CURRENT_FILE variable inside agenskills.py is never reset.

Added in testfile a pytest fixture to call a (newly added) method to reset the CURRENT_FILE variable after each test.
Importing the variable in the testfile and trying to reset it there did **not** work!

Run single test script like:
`poetry run pytest ./tests/unit/test_agent_skill.py`

grafik

### CODE REVIEW

**Review by iFurySt — APPROVED** [review]

**Comment by iFurySt** [comment]
LGTM

File: opendevin/runtime/plugins/agent_skills/agentskills.py [hunk 1]
```diff
@@ -130,6 +130,11 @@ def _cur_file_header(CURRENT_FILE, total_lines):
     return f'[File: {os.path.abspath(CURRENT_FILE)} ({total_lines} lines total)]\n'
 
 
+def init_file():
+    global CURRENT_FILE
+    CURRENT_FILE = None
```
Comment (enyst) [inline opendevin/runtime/plugins/agent_skills/agentskills.py:135]: I wonder if we can do something like this in the fixture / tests?

File: tests/unit/test_agent_skill.py [hunk 2]
```diff
@@ -22,12 +23,21 @@
 )
 
 
+# current file needs to be reset after each test or some
+# tests will fail because they are dependent on the current
+# file being set correctly
+@pytest.fixture(autouse=True)
+def reset_current_file():
+    init_file()
```
Comment (enyst) [inline tests/unit/test_agent_skill.py:31]:
```suggestion
    from opendevin.runtime.plugins.agent_skills import agentskills
    agentskills.CURRENT_FILE = None
    yield
    agentskills.CURRENT_FILE = None
```
What do you think about something like this? It seems to work on my machine

Comment (tobitege) [reply tests/unit/test_agent_skill.py:31]: Ohhh... I think I forgot to prefix it with "agentskills.", doh! 🫨 
I'll give it a go an amend this PR!
Thanks!!

**Comment by tobitege** [comment]
Fixed my previous approach, thanks to @enyst ! 🤗 

grafik

Ready for review.

**Review by enyst — APPROVED** [review]
Thanks!

**Pull request merged**